src/TimeSpans.jl

```diff
@@ -1,6 +1,7 @@
 module TimeSpans
 
 using Dates
+using Statistics
```

Comment:
Typically preferable to only import what is needed
```suggestion
using Statistics: Statistics
```